queries.yaml

```diff
@@ -1740,6 +1741,58 @@ post_process:
       source_layer: buildings
       target_value_type: int
 
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 17
```

Comment:
Consider 16 instead? Let's see the size impact.

Here's zoom 15, so when the smaller building are visible, they can start with a color and grow in size, versus growing in size and then popping in colour:

<img width="779" alt="image" src="https://user-images.githubusercontent.com/853051/153308057-7c382bfb-2411-4ab1-8489-3bb8dd60038f.png">

Reply:
changed to 16 in next commit